src/main/java/com/nexmo/client/verify/endpoints/VerifyEndpoint.java

```
@@ -0,0 +1,142 @@
+ */
+package com.nexmo.client.verify.endpoints;
+
+import com.nexmo.client.HttpWrapper;
+import com.nexmo.client.NexmoClientException;
+import com.nexmo.client.NexmoResponseParseException;
+import com.nexmo.client.auth.TokenAuthMethod;
+import com.nexmo.client.legacyutils.XmlParser;
+import com.nexmo.client.verify.VerifyRequest;
+import com.nexmo.client.verify.VerifyResult;
+import com.nexmo.client.voice.endpoints.AbstractMethod;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.RequestBuilder;
+import org.apache.http.impl.client.BasicResponseHandler;
+import org.apache.http.message.BasicNameValuePair;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.util.Locale;
+
+public class VerifyEndpoint extends AbstractMethod<VerifyRequest, VerifyResult> {
+    private static final Log log = LogFactory.getLog(VerifyEndpoint.class);
+
+    private static final Class[] ALLOWED_AUTH_METHODS = new Class[]{TokenAuthMethod.class};
+
+    private static final String DEFAULT_URI = "https://api.nexmo.com/verify/xml";
+
+    private XmlParser xmlParser = new XmlParser();
+
+    private String uri = DEFAULT_URI;
+
+    /**
+     * Create a new VerifyEndpoint.
+     * <p>
+     * This client is used for calling the verify API's verify endpoint.
+     */
+    public VerifyEndpoint(HttpWrapper httpWrapper) {
+        super(httpWrapper);
+    }
+
+    @Override
+    protected Class[] getAcceptableAuthMethods() {
+        return ALLOWED_AUTH_METHODS;
+    }
+
+    @Override
+    public RequestBuilder makeRequest(VerifyRequest request) throws NexmoClientException, UnsupportedEncodingException {
+        RequestBuilder result = RequestBuilder.post(this.uri)
+                .addParameter("number", request.getNumber())
+                .addParameter("brand", request.getBrand());
```

Is `number` or `brand` required? If so, should we throw an error if they're null?